source/extensions/filters/network/kafka/protocol_code_generator/requests_test_cc.j2

```diff
@@ -0,0 +1,61 @@
+// DO NOT EDIT - THIS FILE WAS GENERATED
```

Comment:
automated tests so serialization changes are actually tested on live data

Reply:
Please add as a comment, also similar comments about where this file ends up being generated to (same elsewhere)